src/templates/post.js

```diff
@@ -111,24 +118,41 @@ const BlogPost = ({ data, children }) => {
     }
   };
 
+  const getActiveTitle = () => {
+    if (typeof window !== 'undefined' && typeof window.document !== "undefined") {
```

Comment:
The window.document is not necessary, but, the window is necessary because if I don't define it different from undefined, the test in Netlify will have an error.